apiserver/facades/client/modelgeneration/interface.go

```diff
@@ -6,22 +6,9 @@ package modelgeneration
 import (
 	"gopkg.in/juju/charm.v6"
 	"gopkg.in/juju/names.v2"
-
-	"github.com/juju/juju/apiserver/params"
-	"github.com/juju/juju/core/model"
 )
 
-//go:generate mockgen -package mocks -destination mocks/package_mock.go github.com/juju/juju/apiserver/facades/client/modelgeneration APIFacade,State,Model,Generation,Application
-
-// APIFacade defines the methods exported by the model generation API facade.
-type APIFacade interface {
```

Comment:
The original intent was obviously to have a facade mock on-site, but this is not used.

Since changing the API requires updating it (and it was already incongruent with prior changes) I've just removed it.